front-end/package.json

```diff
@@ -15,7 +15,7 @@
     "styled-components": "^5.1.0"
   },
   "scripts": {
-    "start": "PORT=3001 react-scripts start",
```

Comment:
Hmm why are we removing this? Wanted to make it explicit that Front End is run on a different port for those that want to run this locally without Docker

Reply:
Had some trouble initially setting up Docker and had removed this. However, rebuilt it after I got everything working and it works with PORT=3001 there. Added it back.